modules/workbench/modules/reporting/assets/schemas/dataset_domain_id_value.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "description": "Time snapshot was taken, in Epoch milliseconds. Same across all rows and all tables in the snapshot, and uniquely defines a particular snapshot.",
+    "name": "snapshot_timestamp",
+    "type": "INTEGER"
+  },
+  {
+    "name": "dataset_id",
+    "type": "INTEGER",
+    "description": ""
+  },
+  {
+    "name": "domain_id",
+    "type": "STRING",
```

Comment:
should this be an integer?

Reply:
No, it's a string in the source table. 
![Screen Shot 2020-12-08 at 1 27 06 PM](https://user-images.githubusercontent.com/53479492/101525309-20cb5080-3959-11eb-80c4-0060ff6b93c4.png)